run/GCHP/HISTORY.rc.templates/HISTORY.rc.CO2

```diff
@@ -13,10 +13,12 @@ VERSION: 1
 # in the collections section of this file.
 #
 # See the examples below for defining different types of grid at different
-# resolutions.
+# resolutions. See the collections section later on in this file for
+# instructions on using an alternative grid for output.
 #
-# See the collections section later on in this file for instructions on
-# using an alternative grid for output.
+# Beware that the online regridding is area-conserving. Only diagnostics
+# independent of grid cell area should be output on a custom grid. For more
+# information on this see the HISTORY.rc page at readthedocs.org/projects/gchp/.
 #==============================================================================
```

Comment:
@lizziel, the readthedocs.org/projects/gchp link takes you to the GCHP dashboard.  I presume you would want the reader to go to gchp.readthedocs.io.

Ditto for the other files below.

Reply:
Yes, I'll push a fix. Thanks for catching that!